asset/snippets/test/samples.test.js

```diff
@@ -259,4 +291,42 @@ describe('quickstart sample tests', () => {
     await metadataTable.delete();
     await resultsTable.delete();
   });
+
+  it('should get effective iam policies successfully', async () => {
+    const assetName = `//storage.googleapis.com/${bucketName}`;
+    const stdout = execSync(`node getBatchEffectiveIamPolicies ${assetName}`);
+    assert.include(stdout, assetName);
+  });
+
+  it('should create saved query successfully', async () => {
+    const description = 'description';
+    await createSavedQuery(queryId, description);
+    assert.strictEqual(console.log.callCount, 6);
+    assert.include(console.log.firstCall.args, savedQueryFullName);
+  });
+
+  it('should list saved queries successfully', async () => {
+    await listSavedQueries();
+    assert.isTrue(console.log.callCount >= 6);
+  });
+
+  it('should get saved query successfully', async () => {
+    await getSavedQuery(savedQueryFullName);
+    assert.strictEqual(console.log.callCount, 6);
+    assert.include(console.log.firstCall.args, savedQueryFullName);
+  });
+
+  it('should update saved query successfully', async () => {
+    const newDescription = 'newDescription';
+    await updateSavedQuery(savedQueryFullName, newDescription);
+    assert.strictEqual(console.log.callCount, 6);
+    assert.include(console.log.firstCall.args, savedQueryFullName);
+    assert.include(console.log.secondCall.args, newDescription);
+  });
+
+  it('should delete saved query successfully', async () => {
+    await deleteSavedQuery(savedQueryFullName);
+    assert.strictEqual(console.log.callCount, 1);
```

Comment:
Overkill, _unless_ you want to make sure at **most** one `console.log` call happened.

Reply:
removed. changed it to "Delete successfully" as discussed offline.